app/dashboard/static/js/app/view-tests-board-job.2017.7.2.js

```diff
@@ -0,0 +1,411 @@
+                gBatchCountMissing[elementId] = count;
+            }
+        }
+    }
+
+    function getSuitesCountFail() {
+        html.replaceByClassHTML('count-badge', '&infin;');
+    }
+
+    function getSuitesCountDone(response) {
+        var results;
+
+        function _parseOperationsResult(result) {
+            gTableCount[result.operation_id] =
+                parseInt(result.result[0].count, 10);
+        }
+
+        function _updateTable(opId) {
+            updateOrStageCount(opId, gTableCount[opId]);
+        }
+
+        results = response.result;
+        if (results.length > 0) {
+            // Parse all the results and update a global object with
+            // the operation IDs and the count found.
+            results.forEach(_parseOperationsResult);
+            // Invalidate the cells in column #3 before updating the DOM
+            // elements. In this way we have the correct 'filter' values in the
+            // global object that we can use to provide the search parameters.
+            gTestsTable.invalidateColumn(3);
+            // Now update the DOM with the results.
+            Object.keys(gTableCount).forEach(_updateTable);
+
+            gTestsTable
+                .search(gSearchFilter);
+        } else {
+            html.replaceByClassTxt('count-badge', '?');
+        }
+    }
+
+    function getSuitesCount(response) {
+        var batchOps;
+        var deferred;
+        var suiteId;
+        var suiteCommit;
+        var queryStr;
+        var queryData;
+        var results;
+
+        batchOps = [];
+
+        function _createOp(result) {
+            suiteId = result._id;
+            suiteCommit = result.vcs_commit;
+
+            if (suiteId) {
+                queryStr = 'test_suite_id=' + suiteId.$oid;
+            }
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: 'cases-total-count-' + suiteCommit,
+                resource: 'count',
+                document: 'test_case',
+                query: queryStr
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: 'cases-success-count-' + suiteCommit,
+                resource: 'count',
+                document: 'test_case',
+                query: queryStr + '&status=PASS'
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: 'cases-fail-count-' + suiteCommit,
+                resource: 'count',
+                document: 'test_case',
+                query: queryStr + '&status=FAIL'
+            });
+
+            batchOps.push({
+                method: 'GET',
+                operation_id: 'cases-unknown-count-' + suiteCommit,
+                resource: 'count',
+                document: 'test_case',
+                query: queryStr + '&status=OFFLINE&status=UNKNOWN'
+            });
+        }
+
+        function _getTestSuiteDone(response) {
```

Comment:
Better to change the variable name here, since `response` is already defined in the outside scope.